### ISSUE TITLE
Create a JS SDK for Hive

### ISSUE THREAD

**Issue by kpachhai** [tuum-tech/profile#10]
Look at the example in JAVA and create a similar SDK but in JS that can be used on the web: https://github.com/elastos/Elastos.NET.Hive.Java.SDK/tree/master/src/main/java/org/elastos/hive

The SDK should be located at https://github.com/tuum-tech/Elastos.NET.Hive.JS.SDK

Check out the Hive backend node code at: https://github.com/elastos/Elastos.NET.Hive.Node

Take a look at the example file https://github.com/BenjaminPiette/elastOS-drafts/blob/master/FacebookHive%2B%2BSample/HyperIMEndpointsExample.md to look at the examples on how to interact with the Hive node.

We're basically building a wrapper for a lot of API endpoints for this first version

For reference on how the Hive plugin works, check out https://github.com/elastos/Elastos.Trinity.Plugins.Hive

In short, this SDK should be easy to implement because all we're doing is interacting with a backend REST API endpoint. The authentication will be done directly on the website frontend(not via this SDK)
